### DISCUSSION TITLE
Roadmap question: QueryCondition implementation?

### DISCUSSION THREAD

**Discussion by BalefulGargoyle** [eProsima/Fast-DDS#3644] [Q&A]
Hello,

I am considering to use Fast DDS, but I require the use of QueryConditions and I have seen that they are not yet implemented. Is that a feature I can expect soon?

Thank you and kind regards.

**Comment by JLBuenoLopez** [comment]
Hi @BalefulGargoyle,

QueryCondition is not in the current [Fast DDS roadmap](https://github.com/eProsima/Fast-DDS/blob/master/roadmap.md) but you might be interested in contacting our [commercial support team](https://github.com/eProsima/Fast-DDS/tree/master#getting-help) about options of including this feature in Fast DDS roadmap.
